erpnext/patches/v7_1/update_component_type.py

```diff
@@ -0,0 +1,13 @@
+import frappe
+from frappe.utils import flt
+
+def execute():
```

Comment:
This patch can be written in one line:

frappe.db.sql("""update `tabSalary Component` set type=if(parentfield='earnings', 'Earning', 'Deduction')""")